tasktiger_admin/templates/tasktiger_admin/tasktiger_task_detail.html

```diff
@@ -1,39 +1,128 @@
 {% extends 'admin/index.html' %}
-{% block body %}
 
-<h2><a href="../../../">TaskTiger</a> – <a href="../">{{ queue }} ({{ state }})</a></h2>
+{% block head %}
+    {{ super() }}
+    <style type="text/css">
+        th {
+            width: 1px;
+            white-space: nowrap
+        }
+    </style>
+{% endblock %}
 
-<pre>{{ task_dumped }}</pre>
-{% if integrations %}
-    <p>Links:</p>
-    <ul>
-    {% for name, url in integrations %}
-        <li><a href="{{ url }}">{{ name }}</a></li>
-    {% endfor %}
-    </ul>
-{% endif %}
-{% if executions_dumped %}
-<p>Executions:</p>
+{% block body %}
 
-    <ul>
+    <h2><a href="../../../">TaskTiger</a> – <a href="../">{{ queue }} ({{ state }})</a></h2>
 
-        {% for execution_dumped, traceback, execution_integrations in executions_dumped %}
+    <table class="metrics table table-striped table-bordered">
+        <thead>
 
-        <li><pre>{{ execution_dumped }}
+        </thead>
+        <tbody class="searchable">
+        <tr>
+            <th>id</th>
+            <td>{{ task_data.id }}</td>
+        </tr>
+        <tr>
+            <th>func</th>
+            <td>{{ task_data.func }}</td>
+        </tr>
+        <tr>
+            <th>args</th>
+            <td>{{ task_data.args }}</td>
+        </tr>
+        <tr>
+            <th>kwargs</th>
+            <td>{{ task_data.kwargs }}</td>
+        </tr>
+        <tr>
+            <th>time last queued</th>
+            <td>{{ task_data.time_last_queued.strftime("%Y-%m-%d %H:%M:%S") }}</td>
+        </tr>
+        <tr>
+            <th>dump</th>
+            <td>
+                <details>
```

Comment:
Trying this fancy html attribute for toggle, should be fairly supported and enough for the admin page.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details#browser_compatibility